src/Features/CSharp/Portable/Completion/CompletionProviders/OverrideCompletionProvider.cs

```diff
@@ -165,22 +173,46 @@ public override bool TryDetermineReturnType(SyntaxToken startToken, SemanticMode
                         break;
                     default:
                         // Anything else and we bail.
-                        return false;
+                        if (SyntaxFacts.IsKeywordKind(token.Kind()))
+                        {
+                            return false;
+                        }
+
+                        break;
```

Comment:
can we be more specific in terms of what tokesn we're allowing breaking here.  it looks like virtually anything.

Reply:
The original code was there to filter out invalid modifiers for `override` completions (e.g. `static override`). I kept this logic with the condition above. I cannot think of what "virtually anything" can be exept attribute tokens. But even if I am missing something, the `break` doesn't matter much as it would just not pass conditions after the enclosing `while` loop